planner/core/rule_aggregation_push_down.go

```diff
@@ -287,6 +308,16 @@ func (a *aggregationPushDownSolver) checkAnyCountAndSum(aggFuncs []*aggregation.
 	return false
 }
 
+func (a *aggregationPushDownSolver) checkAllArgsColumn(fun *aggregation.AggFuncDesc) bool {
```

Comment:
Can you add a comment:
`Returns true if all the arguments of the aggregate function are columns (e.g. sum(a)), instead of expressions (e.g. sum(a+1))`